Doc/library/multiprocessing.rst

```diff
@@ -2198,10 +2198,10 @@ with the :class:`Pool` class.
       (see :meth:`object.__del__` for more information).
 
    .. versionadded:: 3.2
-      *maxtasksperchild*
+      *maxtasksperchild*.
```

Comment:
Adding a full stop here seems odd, as it is just noting when the parameter was added. But perhaps it is better style?

Reply:
17 out of the 19 occurrences of "New in version" in the docs for this module is followed by a space and either "x.x." or "x.x: \<some text\>." where "x" is some number. I suggest adding a period for these other two occurrences so that there is consistency. I am happy to delete the suggested periods if you would like me to.

Reply:
The comment was more me wondering aloud, please keep as-is.

A